sam2/sam_manager/sam_manager.py

```diff
@@ -0,0 +1,170 @@
+import os
+import sys
+import torch 
+import logging
+
+# Add SAM2 to path
+SAM2_PATH = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+sys.path.insert(0, SAM2_PATH)
+
+from sam2.build_sam import build_sam2_video_predictor
+
+from ..server.utils import *
+log = logging.getLogger('myapp')
```

Comment:
Because this project is meant to be used with others, we can give it a specific name in the logging in case we aggregate all log later.